src/components/button/index.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import classes from './styles.module.css';
+import SnackBar from '../snackbar/index';
+import {isEmptyString} from '../utils/index';
+
+const Button = props =>{
+    const {validity,inputTextVal,handleInputTextChange} = props;
+    const [openSnackbar,setOpenSnackbar]=useState(false);
+    const [snackbarMessage,setSnackbarMessage]=useState('');
+    const [snackbarSeverity,setSnackbarSeverity]=useState('');
+
+    const handleButtonClick=()=>{
+        if(isEmptyString(inputTextVal)){
+            setSnackbarMessage('Please Enter text to Encrypt!');
+            setSnackbarSeverity('info');
+            setOpenSnackbar(true);
+        }
+        else{
```

Comment:
better to use ternary operators
const message = isEmptyString(inputTextVal) ? 'Please enter text to encrypt' : 'Successfully encryped the message';
setSnackbar(message);

similar thing can be done for severity

Reply:
Done!